responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -321,8 +257,83 @@ def _whatif(self, id, X, X_feature_new, feature_name, y, alpha=0.1):
                               y, alpha=alpha).to_dict(orient="records")
 
     def compute(self):
-        """No-op function to comply with model analysis design."""
-        pass
+        """Computes the causal effects by running the causal
```

Comment:
it looks like the documentation on the add() function needs to be updated, as it no longer can return a result, so that doc needs to be removed I think?

Reply:
Done.